alvr/server/cpp/alvr_server/OvrHMD.h

```diff
@@ -68,10 +69,14 @@ class OvrHmd : public vr::ITrackedDeviceServerDriver
 
 	void updateIPDandFoV(const TrackingInfo& info);
 
+	bool IsTrackingRef() const { return m_deviceClass == vr::TrackedDeviceClass_TrackingReference; }
+	bool IsHMD() const { return m_deviceClass == vr::TrackedDeviceClass_HMD; }
```

Comment:
`isHMD` in the `OvrHMD` class seems odd.
This class will need some refactoring at some point, but don't do that right now, we can live with the oddity for the moment.

Reply:
Yes it is a bit odd i agree, initially I did consider refactoring this myself too, into 2 classes, a base class that creates a tracked device of the tracking ref class without the parts that deal with render context setup, encoding, and video streaming, etc and a derived version with those parts and resetting the device class to hmd. I went for the less drastic change, I don't like it myself but I wasn't familiar with the code base at the time and/or the situation with the Linux port. If you would like me to do this refactor in this or similar way I'm happy to do it.

Reply:
Just leave it for the time being, we can move things later if we want to.